test/Test/P/Applicative.hs

```diff
@@ -21,6 +29,25 @@ prop_orEmpty i =
      (orEmpty (Just s) === Just s) .&&.
      (orEmpty (Nothing :: Maybe (Sum Int)) === Just (Sum 0))
 
+prop_applicative_monoid = eq $ prop_Monoid (T :: T (ApplicativeMonoid Identity (Sum Int)))
+
+newtype Identity a =
+  Identity { unIdentity :: a }
+  deriving (Show, Eq, Functor)
```

Comment:
Ah great I thought it was in transformers too. I didn't want to add it as a dependency.